projectTemplates/js-server/package.json

```diff
@@ -43,6 +44,7 @@
     "eslint-plugin-flowtype": "^4.7.0",
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-jsx-a11y": "^6.3.1",
+    "eslint-plugin-markdown": "^2.0.0-rc.0",
```

Comment:
is there a version that is not rc? we should stick to stable

Reply:
changed to latest 1.0.2